How-To-Set-Up-Chocolatey-Server.md

```diff
@@ -203,6 +204,46 @@ To configure for performance, you will want to do the following:
 We are looking to add support for the package source to automatically handle this aspect - http://blog.nuget.org/20150922/Accelerate-Package-Source.html
 
 
+### Configure Simple Server alongside WSUS admin-site
+
+If you are running the Simple Server next to a WSUS admin-site, you need to change some settings in the IIS-Configuration for making the Simple Server work correctly again.
+This is due to the fact that the WSUS admin-site installation registers two modules in the IIS root configuration, which will be passed down to all other sites running on that IIS including the Chocolatey Simple Server site.
+
+So for making the chocolatey simple server work again you need to disable inheritance for these modules in order to remove them from the chocolatey Simple Server site.  
+
+*Before making changes to your ISS configuration, read all steps carefully and make sure that you have a backup*  
```

Comment:
```suggestion
*Before making changes to your IIS configuration, read all steps carefully and make sure that you have a backup*  
```

Reply:
OK...apparently Chocolatey has not made it into space (yet ;) )